test/Microsoft.NET.TestFramework/Commands/GetValuesCommand.cs

```diff
@@ -87,10 +87,6 @@ public List<string> GetValues()
             string outputFilename = $"{_valueName}Values.txt";
             var outputDirectory = GetOutputDirectory(_targetFramework, Configuration ?? "Debug");
 
-            outputDirectory.Should().OnlyHaveFiles(new[] {
-                outputFilename,
-            });
```

Comment:
I'm removing this check so that you can do a normal build (which will put other files in the output folder) and then get the values for specific items or properties.  I think I originally put this check in to make sure that the GetValuesCommand wasn't unintentionally building the project, but I don't think there's much value in doing so.